charts/humio-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: humio-operator
-version: 0.17.0
+version: 0.18.0
```

Comment:
Not sure what's the policy regarding bumping the chart version. Are you keeping it in sync with the appVersion bellow?

What we are doing here is adding a new feature in a backward compatible way, so according to semver the we should bump from 0.17.0->0.18.0.

Reply:
It's mostly kept in sync, yes. It doesn't need to strictly be the same version of the image but we found it to be confusing if they drift.

I think we'd want to use the version that is present prior to merging this PR. We'll bump this version on the next release.

Reply:
Sure! I reverted this change: the chart version is now back to `0.17.0`. I will leave the versionning up to you, depending on your release process and schedule.